src/ssl.c

```diff
@@ -12451,6 +12487,7 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD *md)
             return NULL;
     #endif
 
+        XMEMSET(hmac, 0, sizeof(Hmac));
```

Comment:
If this is a new requirement, MEMSETing before use, then a customer may get caught by this.
Should we add an init function?